apps/e2ee-server/.yarnrc.yml

```
@@ -0,0 +1 @@
+nodeLinker: node-modules
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)